client/debug.log

```diff
@@ -1,2 +1,3 @@
 [0116/110039.993:ERROR:directory_reader_win.cc(43)] FindFirstFile: ������� �� ������� ����� ��������� ����. (0x3)
```

Comment:
You need to add npm-debug.log* to your .gitignore file, because the current status of the .gitignore file means npm-debug.log only.

https://stackoverflow.com/questions/56830703/npm-debug-log-is-in-gitignore-file-but-i-still-get-the-messsage-nothing-added